chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/SchedulerCertifiedProductSearchDetailsAsync.java

```diff
@@ -19,7 +19,8 @@
  */
 @Service("schedulerCertifiedProductSearchDetailsAsync")
 public class SchedulerCertifiedProductSearchDetailsAsync {
-    private static final Logger LOGGER = LogManager.getLogger(SchedulerCertifiedProductSearchDetailsAsync.class);
+    //private static final Logger LOGGER = LogManager.getLogger(SchedulerCertifiedProductSearchDetailsAsync.class);
+    private Logger logger;
```

Comment:
Remove the commented out line? Just curious, is there a reason for moving all the static loggers into the getLogger method? Most of the rest of our code uses the static logger initialization.

Reply:
This was done to allow for each job to log to it's own file.  For classes that are shared by multiple jobs, the "job"  class will pass the correct logger to the shared class.  If no logger is set, the getter will initialize the basic logger for the class.

Reply:
So this particular class is not an actual "job".  I consider it a "helper/shared" class.  Job classes should have a the normal static logger defined.  Helper/shared classes should have the logger passed in so that they use the same logger as the job that uses the helper/subordinate class.